.github/workflows/release.yml

```diff
@@ -83,6 +83,9 @@ jobs:
     - name: Publish packages to npmjs
       if: github.event.inputs.pushToNpmJs == true 
       run: yarn publish:next
+  notify:
```

Comment:
Shouldn't the `notify` job depend on the `build` job? Jobs run in parallel by default.
For running sequentially - https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idneeds

Reply:
yes correct @azatsarynnyy